doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
+.. _plutus_tools_component_descriptions:
+
+Plutus tools component descriptions
+=================================================
+
+The Plutus team has provided a set of “Tools”---components, libraries and 
+executables---for external developers to use. They are located in the 
+`plutus-apps <https://github.com/input-output-hk/plutus-apps>`_ repository. 
+
+For each tool listed below, we have indicated its specific location within 
+plutus-apps, provided a short description, indicated its dependencies, and given 
+a brief statement about its state of development to give you some context. 
+
+1. Plutus script utils
+--------------------------
+
+`Plutus script utils <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-script-utils>`_ 
+is a utility library for helping users write Plutus scripts that are to be used on-chain. 
+Currently, this is a “kitchen sink” library containing code that was moved out of 
+plutus-ledger-api. 
+
+It provides: 
+
+   * hashing functions for Datums, Redeemers and Plutus scripts for any Plutus language version. 
+   * functionality for wrapping the untyped Plutus script with a typed interface. 
+   * utility functions for working with the ScriptContext of a Plutus Script. 
+
++--------------+---------------------------------------------------------------+
+| Plutus Script Utils                                                          |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-script-utils``                           |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api                                |
++--------------+---------------------------------------------------------------+
+| State        | Work in progress with an unstable interface.                  |
++--------------+---------------------------------------------------------------+
+
+2. Marconi
+--------------------------
+
+`Marconi <https://github.com/input-output-hk/plutus-apps/tree/main/marconi>`_ is an 
+application for indexing useful data from the Cardano blockchain in a scalable way. 
+
```

Comment:
We might want to mention that it is intended to eventually replace the chain-index

Reply:
Do we have a precise reason to say "scalable way" here? Marconi is basically a learner version of the chain-index, which makes it faster and more flexible.

Reply:
After checking with Konstantinos, no, "scalable way" is not a required phrase. Emphasizing flexibility is more the focus.